gearcmd/worker.go

```diff
@@ -43,6 +44,11 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 		data["error_message"] = err.Error()
 		data["value"] = 0
 		data["success"] = false
+		if conf.RetryCount > 0 {
+			log.Println(kayvee.FormatLog("gearcmd", kayvee.Error, "RETRY", data))
```

Comment:
Actually, it looks like `ZeroBackoff` doesn't allow customizing the # of retries. It should be pretty easy to implement your own `Backoff` type (implementing the `Backoff` interface) that retries a fixed # of times. It's probably more code than this, but would simplify error handling and logging, I think.